src/logger.cpp

```diff
@@ -0,0 +1,33 @@
+#include "logger.hpp"
+
+
+LogLevel Logger::m_msgLevel = LogLevel::info;
+
+bool Logger::m_isPrint = false;
+
+std::string Logger::GetLabel(LogLevel const & type)
+{
+  std::string label;
+  switch(type)
+  {
+    case LogLevel::trace:
+      label = "trace";
+      break;
+    case LogLevel::debug:
+      label = "debug";
+      break;
+    case LogLevel::info:
+      label = "info";
+      break;
+    case LogLevel::warning:
+      label = "warning";
+      break;
+    case LogLevel::error:
+      label = "error";
+      break;
+    case LogLevel::fatal:
+      label = "fatal";
+      break;
```

Comment:
However, I could argue that due to the fact that I use _enum class_ I won't be able to pass any parameter except the _LogLevel_ so the _default_ only handle a case when a programmer make a mistake inside the _std::string Logger::GetLabel(LogLevel const & type)_, but it's hard to make it, because the function is simple. In any other case compiler will warn you)